js/controller.js

```diff
@@ -453,6 +455,16 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       this.state.screenToShow = CONSTANTS.SCREEN.INITIAL_SCREEN;
     },
 
+
+    isChromecastEnabled: function (params) {
+      var chromecastConfig = params.chromecast;
+      var appId = Utils.getPropertyValue(chromecastConfig, 'appId', '');
```

Comment:
What id appID is undefined or null, will getPropertyValue returns and not crashing? And is it returning the right value?